client/state/selectors/get-blocked-sites.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { map, pickBy } from 'lodash';
+
+/**
+ * Returns a list of site IDs blocked by the user
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array}        Blocked site IDs
+ */
+export default function getBlockedSites( state ) {
+	return map( Object.keys( pickBy( state.reader.siteBlocks.items ) ), Number );
```

Comment:
I'm not sure if I'd just want to return just `state.reader.siteBlocks.items`, because that includes sites that _were_ blocked but aren't currently, e.g.

`12345: false`

...hence the `pickBy`. 

With those removed, it felt nicer to return

`[ 123, 124, 125 ]`

from the selector rather than

`[ { 123: true }, { 124: true }, { 125: true } ]`

because the `true` is redundant - they're all blocked :)